catalogs/README.md

```diff
@@ -6,7 +6,7 @@ This folder contains a series of scripts used to generate STAC Catalogs for this
 
 | Catalog Name | Folder | Published Location |
 |--------------|--------|--------------------|
-| AVIRIS       | aviris | `s3://aviris-data/stac-catalog/catalog.json` |
+| AVIRIS       | aviris | `s3://aviris-data/stac-catalog/collection.json` |
```

Comment:
@CloudNiner can we still add a single `catalog.json` file that would have a reference to a single collection?

Reply:
This generated some internal discussion that led to https://github.com/radiantearth/stac-spec/issues/925

For now I've reverted to a single Collection within a Catalog at the root for consistency with other catalogs such as https://landsat-stac.s3.amazonaws.com/catalog.json

@pomadchin if this looks good I'll merge and begin the reupload and reingest into Franklin